### PR TITLE
Fix science shuttle+dock

### PR DESCRIPTION
**Pull request by Ohagi-Chan**
## About The Pull Request
fixes some mapping errors on the science outpost shuttle and its station-side dock.  some other changes were made to better accomodate the new shape of the shuttle

## Why It's Good For The Game
looks a little better

## Changelog
:cl:
tweak: changed the shape of the science outpost shuttle
tweak: changed the shape of the science outpost shuttle station dock
/:cl:

### CODE REVIEW

**Comment by Ohagi-Chan** [comment]
when I tried to add change notes to the map it caused the map to be unreadable  (I believe this is because I can't add comments between the curly brackets in the later stages of the file when it is placing all of the object keys onto the cartesian grid.)

**Comment by TheFakeElon** [comment]
Yeah, don’t touch map file directly. Instead make a copy of the shuttle and give it “austation” somewhere in its name. Change the code reference from the normal one to that and viola, you can now comment that line

**Comment by Ohagi-Chan** [comment]
> Yeah, don’t touch map file directly. Instead make a copy of the shuttle and give it “austation” somewhere in its name. Change the code reference from the normal one to that and viola, you can now comment that line

Thanks, I will give that a shot tonight.

**Comment by Ohagi-Chan** [comment]
Is there anything that needs to be changed?

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -830,13 +830,14 @@
...
 /area/mine/laborcamp)
+//  shuttle changed to science/aus because the bee shuttle is poopy
 "du" = (
 /obj/docking_port/stationary{
 	dwidth = 6;
 	height = 6;
 	id = "science_outpost";
 	name = "outpost shuttle dock";
-	roundstart_template = /datum/map_template/shuttle/science;
+	roundstart_template = /datum/map_template/shuttle/science/aus;
```
Comment (MCterra10) [inline _maps/map_files/Mining/Lavaland.dmm:840]: why?
just change the docking port variables in the code, don't make an entirely new one *and then* put it in the lavaland map

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -538,6 +538,12 @@
 	name = "science outpost shuttle"
 	can_be_bought = FALSE
 
+/datum/map_template/shuttle/science/aus
+	port_id = "science"
+	suffix = "outpost_aus"
+	name = "science outpost shuttle"
+	can_be_bought = FALSE
```
Comment (MCterra10) [inline code/datums/shuttles.dm:545]: just modify the suffix var on the original one, map edits are so so much worse than just changing code

**Review by MCterra10 — CHANGES_REQUESTED** [review]

**Comment by Ohagi-Chan** [comment]
I have made the requested changes

**Comment by TheFakeElon** [comment]
Single white space in lavaland dmm, day ruined 😔

**Review by MCterra10 — APPROVED** [review]
looking poggers

**Pull request merged**